lib/mix/test/test_helper.exs

```diff
@@ -123,23 +120,27 @@ defmodule MixTest.Case do
 
   def ensure_touched(file, current) do
     File.touch!(file)
+
     unless File.stat!(file).mtime > current do
       ensure_touched(file, current)
     end
   end
 
   def os_newline do
-    case :os.type do
+    case :os.type() do
       {:win32, _} -> "\r\n"
       _ -> "\n"
     end
   end
 
   def mix(args, envs \\ []) when is_list(args) do
-    System.cmd(elixir_executable(),
-               ["-r", mix_executable(), "--" | args],
-               stderr_to_stdout: true,
-               env: envs) |> elem(0)
+    System.cmd(
+      elixir_executable(),
+      ["-r", mix_executable(), "--" | args],
```

Comment:
Can you please do:

    args = ["-r", mix_executable(), "--" | args]

In the previous line so everything fits on the same line? Thank you!

Reply:
Yes, good catch!